pkg/kubectl/resource_printer.go

```diff
@@ -504,6 +504,10 @@ func translateTimestamp(timestamp util.Time) string {
 }
 
 func printPod(pod *api.Pod, w io.Writer, withNamespace bool, wide bool, showAll bool, columnLabels []string) error {
+	return printPodBase(pod, w, withNamespace, wide, showAll, true, columnLabels)
+}
+
+func printPodBase(pod *api.Pod, w io.Writer, withNamespace bool, wide bool, showAll bool, showIfTerminating bool, columnLabels []string) error {
```

Comment:
I didn't see showAll before - what does it do differently than showIfTerminating?